org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -493,6 +493,91 @@ package lang {
 	}
 	
 	/**
+	 * Represents a set of key -> values
+	 * 
+	 */
+	class Dictionary {
+	
+		constructor() { }
+		/**
+		 * Adds or updates a value based on a key
+		 */
+		method put(_key, _value) native
+		
+		/*
+		 * Returns the value to which the specified key is mapped, or null if this Dictionary contains no mapping for the key.
```

Comment:
This implies that the dictionary can not differentiate not having a key from associating a null value with that key. I do not say it is wrong, just bring it up to be sure that it is what we mean.